app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -746,13 +746,18 @@ protected function _vcl_get_probe($probeUrl) {
             .probe = {
                 .request =
                     "GET {{probe_path}} HTTP/1.1"
-                    "Host: {{probe_host}}"
+                    "Host: {{probe_host}}"{{probe_https}}
```

Comment:
How will this impact users not using https? Do you foresee any backward compatibility issue?